navigator-menu/src/components/tips-calculator-components/right-portion/amount-component/Amount.jsx

```diff
@@ -0,0 +1,25 @@
+import { Wrapper, Parag, Strong, H1 } from './Amount.styles';
+import PropTypes from 'prop-types';
+
+function Amount({ name, amount }) {
+  return (
+    <Wrapper>
+      <Parag>
+        <Strong>{name}</Strong>
+        <br />/ person
+      </Parag>
+      {amount === 'NaN' ? (
```

Comment:
This is a bad check. Research on how to make sure a JavaScript object has a numerical value.